README.md

```diff
@@ -1,5 +1,97 @@
 # e-commerce-app
 
-## My first Pull!
+This is a responsive app of an online clothing store.
+The user can see products from various categories of clothes, he can mark favourite items he liked or put them straight in the shopping cart and purchase it.
 
-# My second Push
+## App pages
+
+---
+
+-   **Homepage**
+
+    _This is the main screen of the site where the user will be exposed to the best-selling items, the recommendations of the stylists and will also be able to see the variety of existing categories and choose to view items in a specific category._
+
+---
+
+-   **Registration page**
+
+    _This page accepts new user credentials and creates a new use in the database._  
+    _It is possible for the user to register via his Facebook or Gmail account._  
+    _After registration the user will be redirected to the login page._
+
+    _Please note that the user can choose not to register and still view the site, but if he wants to make a purchase he will have to register first._
+
+---
+
+-   **Login page**
+
+    _This page is responsible for connecting a registered user to the site by checking his details on database._
```

Comment:
Maybe try this structure for the sections:

## Login
This is the login page.
On this page: 
1. The user can log-in to the application (store). 
2. etc..

(only logged users can make purchases)